test/e2e/specs/specs-playwright/wp-gutenboarding__happy-path.ts

```diff
@@ -63,6 +69,26 @@ describe( DataHelper.createSuiteTitle( 'Gutenboarding: Create' ), function () {
 			] );
 		} );
 
+		it( 'Navigated to Site Editor', async function () {
+			await page.waitForURL( /.*\/site-editor\/.*/, { waitUntil: 'networkidle' } );
+
+			// {@TODO} This is temporary while the FSE spec is awaiting migration to Playwright.
+			const gutenbergEditorPage = new GutenbergEditorPage( page );
+			await gutenbergEditorPage.forceDismissWelcomeTour();
+			const outerFrame = await gutenbergEditorPage.getEditorFrame();
+
+			// There's another iframe within the parent iframe.
+			const innerFrame = ( await (
+				await outerFrame.waitForSelector( 'iframe[name="editor-canvas"]' )
+			 ).contentFrame() ) as Frame;
+			await innerFrame.waitForSelector( '[aria-label="Block: Site Title"]' );
+			const elementHandle = await innerFrame.waitForSelector( '[aria-label="Block: Site Title"]' );
+			const siteEditorTitle = await elementHandle.innerText();
+			expect( siteEditorTitle ).toStrictEqual( siteTitle );
```

Comment:
I've been going back and forth on whether to leverage Jest's built-in `Expect` library to perform assertions where possible.

Advantages of using Expect:
- potentially quicker turnaround; instead of waiting up to 30s for the timeout with `waitForSelector`, `expect` would perform its action in milliseconds.
- encourage pattern of using the 'right tool for the job' so to speak.

Disadvantage of using Expect:
- makes test steps longer.

I think in this case let's try using Expect!